pkg/sdn/plugin/pod.go

```diff
@@ -166,39 +179,105 @@ func (m *podManager) handleCNIRequest(request *cniserver.PodRequest) ([]byte, er
 	return result.Response, result.Err
 }
 
+type runningPodsSlice []*runningPod
+
+func (l runningPodsSlice) Len() int           { return len(l) }
+func (l runningPodsSlice) Less(i, j int) bool { return l[i].ofport < l[j].ofport }
+func (l runningPodsSlice) Swap(i, j int)      { l[i], l[j] = l[j], l[i] }
+
+// FIXME: instead of calculating all this ourselves, figure out a way to pass
+// the old VNID through the Update() call (or get it from somewhere else).
```

Comment:
We generate the Update calls ourselves (and we always will generate them ourselves, regardless of whether CNI ever gets an UPDATE action, because Kubernetes doesn't know about NetNamespaces), so we can include that information if we want to.